zipline/assets/assets.py

```diff
@@ -27,112 +25,87 @@
 
 from zipline.errors import (
     ConsumeAssetMetaDataError,
-    InvalidAssetType,
     MultipleSymbolsFound,
     RootSymbolNotFound,
-    SidAssignmentError,
     SidNotFound,
     SymbolNotFound,
     MapAssetIdentifierIndexError,
 )
-from zipline.assets._assets import (
+from zipline.assets import (
     Asset, Equity, Future
 )
+from abc import ABCMeta
 
-log = Logger('assets.py')
-
-# Expected fields for an Asset's metadata
-ASSET_FIELDS = [
-    'sid',
-    'asset_type',
-    'symbol',
-    'root_symbol',
-    'asset_name',
-    'start_date',
-    'end_date',
-    'first_traded',
-    'exchange',
-    'notice_date',
-    'expiration_date',
-    'contract_multiplier',
-    # The following fields are for compatibility with other systems
-    'file_name',  # Used as symbol
-    'company_name',  # Used as asset_name
-    'start_date_nano',  # Used as start_date
-    'end_date_nano',  # Used as end_date
-]
-
-
-# Expected fields for an Asset's metadata
-ASSET_TABLE_FIELDS = [
-    'sid',
-    'symbol',
-    'asset_name',
-    'start_date',
-    'end_date',
-    'first_traded',
-    'exchange',
-]
-
-
-# Expected fields for an Asset's metadata
-FUTURE_TABLE_FIELDS = ASSET_TABLE_FIELDS + [
-    'root_symbol',
-    'notice_date',
-    'expiration_date',
-    'contract_multiplier',
-]
-
-EQUITY_TABLE_FIELDS = ASSET_TABLE_FIELDS
+from zipline.assets import (
```

Comment:
I don't think so, as zipline.assets is the *package*. Within this package zipline/assets/assets.py imports from zipline/assets/asset_writer.py, and zipline/assets/asset_writer.py imports from zipline/assets/_assets.pyx.